src/cavendish_particle_tracks/_decay_angles_dialog.py

```diff
@@ -55,30 +53,83 @@ def __init__(self, parent=None):
         ):
             self.layout().addWidget(table_datum, i % 3 + 2, i // 3)
 
-        self.layout().addWidget(bss, 4, 0, 1, 3)
-
+        self.layout().addWidget(btn_calculate, 5, 0, 1, 3)
         self.layout().addWidget(
             QLabel("Opening angles"),
-            5,
+            6,
             0,
             1,
             3,
         )
```

Comment:
Good old sensible formatting 🫠 .

If you delete the final comma and run `pre-commit` will it put it all on one line now?